src/Etterna/Models/Misc/XMLProfile.cpp

```diff
@@ -484,6 +438,9 @@ XMLProfile::LoadScreenshotDataFromNode(const XNode* pScreenshotData)
 	ASSERT(pScreenshotData->GetName() == "ScreenshotData");
 	FOREACH_CONST_Child(pScreenshotData, pScreenshot)
 	{
+		if (pScreenshot->GetName() != "Screenshot")
```

Comment:
What is this for? It doesn't look like this was being logged before.

Reply:
This was removed in `0355a8d`, and found a way in again during the rebase. Removed.